examples/running-tests-using-travis-and-saucelabs/README.md

```diff
@@ -0,0 +1,20 @@
+# Example: Running Tests Using Travis CI and Sauce Labs
```

Comment:
Can we follow README style that we have for https://github.com/DevExpress/testcafe/blob/master/examples/basic/README.md ?